common/cpp/react/renderer/components/rnsvg/RNSVGImageShadowNode.h

```diff
@@ -48,7 +48,7 @@ class JSI_EXPORT RNSVGImageShadowNode final : public ConcreteViewShadowNode<
       ShadowNodeFamilyFragment const &familyFragment,
       ComponentDescriptor const &componentDescriptor) {
     auto imageSource = ImageSource{ImageSource::Type::Invalid};
-    return {imageSource, {imageSource, nullptr}};
+    return {imageSource, {imageSource, nullptr, SharedFunction<>(nullptr)}};
```

Comment:
Adding this code will result in RN `0.71` not working on new arch, right? Also, is such code enough:

```suggestion
    return {imageSource, {imageSource, nullptr, {}}};
```

?

Reply:
Hi, I tested with the suggested change and it was enough to solve the issue (I ran into it today when trying the new arch). As to whether it will affect RN `0.71` or not I don't know 🙂

Reply:
On my way testing RN 0.71.

Can also confirm `{}` works (on 0.72)

Reply:
@WoLewicki You're correct on both things

RN 0.71 fails as the constructor is different

https://github.com/facebook/react-native/blob/49d16d5aefd703ff20ab5d05544d78d6ce446427/ReactCommon/react/renderer/imagemanager/ImageRequest.h#L26-L33
```CPP
class ImageRequest final {
 public:
  /*
   * The default constructor
   */
  ImageRequest(
      ImageSource imageSource,
      std::shared_ptr<const ImageTelemetry> telemetry);
```

I suppose things could be addressed by using the processor to check for react-native version.
I'm not entirely sure how to do that though.

Reply:
I think we will just drop the support for 0.71 then. We keep only the newest version working on new arch until it becomes more stable.

Reply:
Sure yeah that's all fine by me. It looks like there is a lot of trouble < 0.72 across the ecosystem anyway.